lib/components/src/blocks/Story.tsx

```diff
@@ -34,12 +35,16 @@ type IFrameStoryProps = CommonProps;
 
 type StoryProps = InlineStoryProps | IFrameStoryProps;
 
-const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id }) => (
+const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id = uuidv4() }) => (
```

Comment:
Can id be undefined ? Seems weird ?

Appart from this, I don't think you need uuid for this. This adds another dependency for something _simple_. Interesting [reading](https://medium.com/javascript-in-plain-english/you-might-not-need-uuid-v4-for-generating-random-identifiers-89e8a28a7d77) about it
`Math.round(Math.random() * 1e5)` will be enough, unless someone plans to have that many story, we can take the risk of having the same id twice :)

Reply:
Yes, it can be undefined, using InlineStory directly, that's why!
Really interesting reading and I'm not against removing this extra dependency (even if it's already used in other packages)

Reply:
Thanks for the explanation ;) 
I'm sure we can apply the same logic for other use of uuid.